.github/workflows/test.yml

```diff
@@ -20,19 +20,23 @@ jobs:
             graphql: '~> 1.12.0'
             client_id: 'false'
             interpreter: yes
+            anycable_rails: '~> 1.2'
           - ruby: 2.7
             graphql: '~> 1.12.0'
             client_id: 'false'
             interpreter: yes
+            anycable_rails: '~> 1.2.0'
           - ruby: 2.6
             graphql: '~> 1.11.0'
             client_id: 'true'
             interpreter: no
+            anycable_rails: '~> 1.0'
```

Comment:
Shouldn't it be `~> 1.0.0` to test older anycable versions?

Reply:
The older version would be picked up automatically due according to the Ruby version